cdap-spark-core-base/src/main/java/co/cask/cdap/app/runtime/spark/classloader/SparkRunnerClassLoader.java

```diff
@@ -145,7 +135,7 @@ public URL apply(String resourceName) {
    */
   @Nullable
   private InputStream openResource(String resourceName) throws IOException {
-    URL resource = findResource(resourceName);
+    URL resource = getResource(resourceName);
```

Comment:
is the change needed here, too? For the rewriter, I get it. But this would affect all classes that are loaded?

Reply:
This is the same as for the rewriter. This one open the stream to the class byte code for the rewriter to use, while the function passed to the rewriter is for the rewriter to further lookup other resources.